api/py/test/test_compile.py

```diff
@@ -307,6 +307,24 @@ def test_compile_feature_display():
     assert result.exit_code == 0
 
 
+def test_table_display_staging_query():
+    """
+    Test a staging query compile produces related table
+    """
+    runner = CliRunner()
+    input_path = f"staging_queries/sample_team/sample_staging_query.py"
+    result = runner.invoke(
+        extract_and_convert,
+        [
+            "--chronon_root=test/sample",
+            f"--input_path={input_path}",
+            "--table-display",
+        ],
+    )
+
+    assert "Output StagingQuery Tables" in result.output
```

Comment:
can we verify one of the table names maybe?

Reply:
Done. Chose to extract the entire output and compare it with hard-coded expected output.

Reply:
Thank you.